src/main/webapp/WEB-INF/views/manufacturers/manufacturer_created.jsp

```diff
@@ -0,0 +1,19 @@
+<%@ page contentType="text/html;charset=UTF-8"%>
+<html>
+<head>
+  <style>
+    h1 {
+      text-align: center;
+      margin-top: 200px
+    }
+    h2 {
+      text-align: center;
+    }
+  </style>
+  <title>Manufacturer was added</title>
+</head>
+<body>
+<h1>Manufacturer ${manufacturer} was successfully registered!</h1>
+<h2><a href="${pageContext.request.contextPath}/index">Back to main menu</a></h2>
+</body>
+</html>
```

Comment:
check jsp files for empty lines at the end 
P.S.
you can configure Intellij IDEA to do that automatically

Reply:
ok